### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by marcelgerber**
cc @ingorichter

### CODE REVIEW

**Comment by MarcelGerber** [comment]
Argh, GitHub had some database issues when I opened this PR.
Never did I delete the branch (at least not after filing the PR; I did so before because GitHub was unresponsive and didn't seem to recognize it).

Let's hope it all works now :^)

File: src/nls/de/strings.js [hunk 1]
```diff
@@ -312,6 +313,7 @@ define({
     "CMD_FILE_NEW"                        : "Neue Datei",
     "CMD_FILE_NEW_FOLDER"                 : "Neuer Ordner",
     "CMD_FILE_OPEN"                       : "Öffnen\u2026",
+    "CMD_RECENT_FILES_OPEN"               : "Zuletzt verwendete öffnen\u2026",
```
Comment (ingorichter) [inline src/nls/de/strings.js]: Should we add "Dateien"? Zuletzt verwendete Dateien/Projekte öffnen

Comment (MarcelGerber) [inline src/nls/de/strings.js]: Changed it.

**Comment by ingorichter** [comment]
LGTM. Merging.

**Pull request merged**